pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Rook Authors. All rights reserved.
+Copyright 2020 The Rook Authors. All rights reserved.
```

Comment:
Do not change the copyright on existing files.

Reply:
I was the person that made the file originally in 2020. I think the file I copied the "Copyright" section for this one was made in 2018. Should I still change it?

Reply:
Regardless of who created the file, if you just edit it you must not change the Copyright section.